tests/__mocks/events/name.ts

```diff
@@ -0,0 +1,5 @@
+import { Event } from "../../../src/type/event";
+
+export class name extends Event {
```

Comment:
Maybe rename this event to "normal", to indicate its the "normal" event mock without anything changed.